src/routers/organizationProposalsRouter.ts

```diff
@@ -0,0 +1,18 @@
+import express from 'express';
+import { organizationProposalsHandlers } from '../handlers/organizationProposalsHandlers';
+import { verifyJwt as verifyAuth } from '../middleware/verifyJwt';
+
+const organizationProposalsRouter = express.Router();
+organizationProposalsRouter.get(
```

Comment:
May as well throw a newline before this line (after the `const`), since (1) that seems consistent with the other routers, and (2) that better communications that the constant defined isn't bound in some special way to the first route defined (the `get`).

Reply:
I get why prettier doesn't try to deal with whitespace but darn it I wish it could!

Reply:
I dunno, what would it do here? This feels more like semantic whitespace, which I would hope Prettier *wouldn't* try to make decisions about.